cypress/utils/index.ts

```diff
@@ -0,0 +1,13 @@
+import { FeatureFlag } from '../../src/featureFlags/flags/featureFlags'
+
+export const getTestSelector = (selectorId: string) => `[data-cy=${selectorId}]`
```

Comment:
Avoid using `data-cy` as it requires (this) custom implementation and introduces a second way to do something that's already covered.

Instead, use `data-testid`, and use the [`*ByTestId` selectors](https://testing-library.com/docs/queries/bytestid) from `@testing-library`.

Reply:
I tried using that function and it turns and html element when cypress expects a string